src/CONST/index.ts

```diff
@@ -6787,6 +6788,7 @@ const CONST = {
             TO: 'to',
             CATEGORY: 'category',
             TAG: 'tag',
+            ORIGINAL_AMOUNT: 'originalamount',
```

Comment:
`originalamount` is the column name expected by the BE. We should keep it in this format for now.